calcom/cal.com
# Delete rescheduled booking

## What does this PR do?

Fixes that rescheduled bookings are not deleted anymore for non seat events. 

Also fixes the number of available seats shown on BookingPage. It was taking the number of available seats of the booking that was rescheduled and not of the new slot the users wants to book. 

## Type of change


## How should this be tested?

- Book event 
- reschedule event 
- check and make sure rescheduled event is not in the list of upcoming bookings anymore
